test/demo.sh

```diff
@@ -23,10 +23,23 @@ export FAROS_DRY_RUN=1 # Set to 0 or comment out to send the event to Faros
 # ../faros_event.sh CI \
```

Comment:
why all the tests are commented out?!

Reply:
or these are not tests? what are these for?!

Reply:
example commands one can run. Not sure if they should be in the test directory

Reply:
I committed them for ease of manual testing and/or demoing the CLI. Im not attached to them if we would rather remove them... also fine with putting them in another place.